.gitmodules

```diff
@@ -1,3 +1,12 @@
 [submodule "docs/_build/html"]
 	path = docs/_build/html
 	url = git@github.com:paltman/pymediainfo.git
+[submodule "MediaInfoLib"]
+	path = MediaInfoLib
+	url = https://github.com/MediaArea/MediaInfoLib.git
+[submodule "ZenLib"]
+	path = ZenLib
+	url = https://github.com/MediaArea/ZenLib.git
+[submodule "zlib"]
+	path = zlib
+	url = https://github.com/madler/zlib.git
```

Comment:
Do we really want to build against the git master of all these projects? That doesn't sound very stable.

Reply:
The projects are at their latest release each. This is not configured in .gitmodule but in the "submodule links".